apps/docs/src/app/pages/docs/landing/docs-page.component.html

```diff
@@ -0,0 +1,7 @@
+<ng-doc-root>
+	<ng-doc-navbar>
+		<h3 style="margin: 0" ngDocNavbarLeft>MyAwesomeDoc</h3>
```

Comment:
I would do either of two things:
1. Add a class and move this `style` to a css/scss file for the component (or use a utility class or something);
2. Or add a TODO comment that makes sure you don't forget to remove this when styling the application.

The danger here is that this attribute is going to get forgetten and cause problems later on when styling the page.